main.go

```diff
@@ -28,17 +28,26 @@ import (
 )
 
 func main() {
-	// get the users home dir
-	user, err := user.Current()
-	if nil != err {
-		panic(fmt.Sprintf("Could not read users homedir %v\n", err))
+	// Find users home directory
+	usr, err := user.Current()
+	home := ""
+	if err != nil {
+	    // Fall back to reading $HOME - work around user.Current() not
+	    // working for cross compiled binaries on OSX or freebsd.
+	    // https://github.com/golang/go/issues/6376
+	    home := os.Getenv("HOME")
```

Comment:
Since `home` is already initialized, this should be assigned with `=` not `:=`, otherwise you would create a new `home`variable inside the ìf` block and never change the value of the `home`variable outside the block.